Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -2096,5 +2098,20 @@ inline Vector<ScrollAxis> BuilderConverter::convertScrollTimelineAxis(BuilderSta
     });
 }
 
+inline Vector<ViewTimelineInsets> BuilderConverter::convertViewTimelineInset(BuilderState& state, const CSSValue& value)
+{
+    // While parsing, consumeViewTimelineInset() and consumeViewTimelineShorthand() yield a CSSValueList exclusively.
+    ASSERT(is<CSSValueList>(value));
```

Comment:
CSS Typed OM can give random values, we've had fuzz bugs about this. We should probably just early return with the default value if we don't have the default value.

Reply:
I don't follow, how does the CSS Typed OM relate to this code? My understanding is that the `CSSValue` here stems only from the result of parsing the `view-timeline-inset` longhand or `view-timeline` shorthand. What am I missing?

Reply:
```
let cssStyleValue = CSSMathValue.parseAll('columns', 'auto')[0];
document.body.attributeStyleMap.append('grid-auto-columns', cssStyleValue);
```

This JS goes to the style builder converter directly without going through the parsing code.

Reply:
I see! So, what do you suggest for this specific function?